db/Controllers/api.controller.js

```diff
@@ -1,5 +1,17 @@
 const endpoints = require('../../endpoints.json')
+const {readTopics} = require('../models/api.model')
 
 exports.getApiDocu = (req, res) => {
     res.status(200).send({endpoints})
+}
+
+exports.getTopics = (req, res) => {
+        readTopics().then((topics) => {
+            //console.log(topics)
```

Comment:
Try to avoid pushing up commented out code! Once you've solved the issue the created the need for the console.log(), remove it 😄